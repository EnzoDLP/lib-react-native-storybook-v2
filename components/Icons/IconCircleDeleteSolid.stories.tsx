import type { Meta, StoryObj } from '@storybook/react';
import React from 'react';
import { View } from 'react-native';
import { IconCircleDeleteSolid } from './IconCircleDeleteSolid.component';

const meta = {
  title: 'Icons/IconCircleDeleteSolid',
  component: IconCircleDeleteSolid,
  decorators: [
    (Story) => (
      <View style={{ padding: 16 }}>
        <Story />
      </View>
    ),
  ],
} satisfies Meta<typeof IconCircleDeleteSolid>;

export default meta;

type Story = StoryObj<typeof meta>;

export const Default: Story = {
  args: {
    accessibilityLabel: 'Ic√¥ne supprimer',
    color: '#000000',
    size: 24,
  },
};
