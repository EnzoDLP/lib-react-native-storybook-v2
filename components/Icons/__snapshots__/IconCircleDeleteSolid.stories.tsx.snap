// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icons/IconCircleDeleteSolid Default smoke-test 1`] = `
<div class="css-view-175oi2r"
     style="padding: 16px;"
>
  <svg width="24"
       height="24"
       viewbox="0 0 24 24"
       fill="none"
       aria-label="IcÃ´ne supprimer"
       role="img"
  >
    <g clip-path="url(#clip0_10323_55129)">
      <path d="M12 0a12 12 0 1012 12A12.013 12.013 0 0012 0z"
            fill="#000000"
      >
      </path>
      <path d="M15.828 9.172a1 1 0 00-1.707-.708L12 10.586 9.878 8.464A1 1 0 108.464 9.88L10.586 12l-2.122 2.121a1 1 0 001.414 1.415L12 13.414l2.121 2.122a1 1 0 101.414-1.415L13.414 12l2.121-2.121a1 1 0 00.293-.707z"
            fill="#FFFFFF"
      >
      </path>
    </g>
  </svg>
</div>
`;
